### DISCUSSION TITLE
[Roadmap] v1.6

### DISCUSSION THREAD

**Discussion by AlexanderVolkovTCS** [Tinkoff/investAPI#448] [Announcements]
- Добавление параметра price_type в большинство методов для удобства конвертации пунктов и рублей. Особенно актуально для замещающих облигаций;
- Добавление в GetClosePrice цены вечерней сессии;
- Доработки песочницы: GetSandboxOperationByCursor;
- Новый метод расчета количества доступных для покупки/продажи лотов;
- Добавление order_request_id в PostOrderResponse;
- Добавление параметра timeInForce в PostOrder - сейчас для лимитных заявок доступно только timeInForce = DAY, станет доступно = fillOrKill / fillAndKill;
- Доработки стриминга:
  - добавление новых таймфреймов;
  - добавление трекинга подписок;
  - исправление бага с отписками по одному инструменту;
- Продублировать message из заголовка в контракте ответа PostOrder, продублировать tracking_id в контрактах ответа всех методов.

**Comment by nonamegithub** [comment]
[не увидел](https://github.com/Tinkoff/investAPI/issues/262)

**Comment by x-tinal** [comment]
Обновите в Nuget для C#, пожалуйста. Там версия от 2 мая и для версии 1.4 аж...
